content/zh/blogs/kubesphere-3.1.0-ga-announcement.md

```diff
@@ -0,0 +1,135 @@
+
+2021 年 4 月 29 日，KubeSphere 开源社区激动地向大家宣布，KubeSphere 3.1.0 正式发布！为了帮助企业最大化资源利用效率，KubeSphere 打造了一个以 Kubernetes 为内核的云原生分布式操作系统，提供可插拔的开放式架构，无缝对接第三方应用，极大地降低了企业用户的使用门槛。
+
+KubeSphere v3.1.0 主打 **“延伸至边缘侧的容器混合云”**，新增了对 **“边缘计算”** 场景的支持。同时在 v3.0.0 的基础上新增了 **计量计费**，让基础设施的运营成本更清晰，并进一步优化了在 **“多云、多集群、多团队、多租户”** 等应用场景下的使用体验，增强了 **“多集群管理、多租户管理、可观测性、DevOps、应用商店、微服务治理”** 等特性，更进一步完善交互设计提升了用户体验。
+
+云原生产业联盟撰写的《云原生发展白皮书》提到，万物互联时代加速了云-边协同的需求演进，传统云计算中心集中存储、计算的模式已经无法满足终端设备对于时效、容量、算力的需求，向边缘下沉并通过中心进行统一交付、运维、管控，已经成为云计算的重要发展趋势。
+
+面对这一发展趋势，KubeSphere 与 KubeEdge 社区紧密合作，将 Kubernetes 从云端扩展至边缘，以统一的标准实现了对边缘基础设施的纳管。通过与 KubeEdge 集成，解决了边缘节点纳管、边缘工作负载调度和边缘可观测性等难题，结合 KubeSphere 已有的多集群管理将混合多云管理延伸至边缘侧。
+
+并且，v3.1.0 得到了来自青云 QingCloud 之外的更多企业与用户的贡献和参与，无论是功能开发、功能测试、缺陷报告、需求建议、企业最佳实践，还是提供 Bug 修复、国际化翻译、文档贡献，这些来自开源社区的贡献都为 v3.1.0 的发布和推广提供了极大的帮助，我们将在文末予以特别致谢！
+
+## 解读 v3.1.0 重大更新
+
+KubeSphere 3.1.0 增加了计量计费功能，支持集群、企业空间的多层级与多租户在应用资源消耗的计量与统计。通过集成 KubeEdge，实现应用快速分发至边缘节点。同时还提供了更强大的可观测性能力，如兼容 PromQL、内置主流告警规则、可视化对接钉钉、企业微信、Slack 和 Webhook 等通知渠道。DevOps 的易用性在 3.1.0 也上了一个台阶，例如内置多套常用流水线模板，支持多分支流水线和流水线复制等，关于重大更新详情请查看文末海报。
+
+## 多维度计量计费，让 K8s 运营成本更透明
+
+在企业运营和管理 Kubernetes 容器平台时，通常需要分析资源消耗，查看集群及其中租户的消费账单，洞察资源使用情况，分析基础设施运营成本。
+
+在 KubeSphere 3.1.0 中，可从多个维度来分析平台资源消耗：
+
+- 从集群维度，可查看每个集群资源消耗，深入到节点中分析运行的工作负载，精准规划每个节点中工作负载的资源使用状况。
+- 从企业空间维度，可查看每个企业空间资源消耗，获取企业空间中项目、应用、工作负载的消费账单，分析多租户环境中各个租户的资源使用是否合理。
+
+另外，除了可以通过界面查看和导出数据，KubeSphere 计量计费平台也提供了所有操作的 API。接下来在后续的版本里，会持续加强并构筑端到端完整的计量计费可运营系统。
+
+![](https://cdn.jsdelivr.net/gh/yangchuansheng/imghosting1@main/img/20210429141959.png)
+
+![](https://cdn.jsdelivr.net/gh/yangchuansheng/imghosting1@main/img/20210429141757.png)
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=802751933&bvid=BV17y4y1s7Nj&cid=330926724&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+## 边缘节点管理
+
+[KubeEdge](https://github.com/kubeedge/kubeedge) 是一个开源的边缘计算平台，它在 Kubernetes 原生的容器编排和调度能力之上，实现了 **云边协同**、**计算下沉**、**海量边缘设备管理**、**边缘自治** 等能力。但 KubeEdge 缺少云端控制层面的支持，如果将 KubeSphere 与 KubeEdge 相结合，可以很好解决这一问题，实现应用与工作负载在云端与边缘节点进行统一分发与管理。
+
+这一设想在 v3.1.0 中得以实现，KubeSphere 现已支持 KubeEdge 边缘节点纳管、KubeEdge 云端组件的安装部署、以及边缘节点的日志和监控数据采集与展示。结合 KubeEdge 的边缘自治功能和 KubeSphere 的多云与多集群管理功能，可以实现云-边-端一体化管控，解决在海量边、端设备上统一完成应用交付、运维、管控的需求。
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=587836535&bvid=BV1zB4y1c7oK&cid=330894124&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+
+## 强化微服务治理能力
+
+KubeSphere 基于 [Istio](https://github.com/istio/istio) 提供了金丝雀发布、蓝绿部署、熔断等流量治理功能，同时还支持可视化呈现微服务之间的拓扑关系，并提供细粒度的监控数据。在分布式链路追踪方面，KubeSphere 基于 Jaeger 让用户快速追踪微服务之间的通讯情况，从而更易于了解微服务的请求延迟、性能瓶颈、序列化和并行调用等。
+
+KubeSphere 3.1.0 对微服务治理功能进行了强化，将 Istio 升级到了 1.6.10，支持图形化流量方向检测，图像化方式显示应用流量的流入/流出。同时还支持对 Nginx Ingress Gateway 进行监控，新增 Nginx Ingress Controller 的监控指标。
+
+![](https://cdn.jsdelivr.net/gh/yangchuansheng/imghosting@three/img/20210428221044.png)
```

Comment:
Please do not use personal CDN storage. You can store these images in this repository or our QingStor Object Storage.